src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -1716,7 +1716,15 @@ public InstructorAttributes getInstructor() {
     }
 
     public List<String> getSections() {
-        return sections;
+        List<String> sectionNames = new ArrayList<>();
+        for (String section : bundle.sectionsInCourse()) {
+            if (!section.equals(Const.DEFAULT_SECTION)) {
+                sectionNames.add(section);
+            }
+        }
+        
+        Collections.sort(sectionNames);
+        return sectionNames;
```

Comment:
Good that we are using the data from the bundle. 
Can you do all the processing only once and store it somewhere?